app/views/physical_ticket/index.html.haml

```diff
@@ -0,0 +1,32 @@
+.container
+  .row
+    .col-md-12.page-header
+      %h2
+        Tickets
+      .text-muted
+        Your tickets for the conference
+
+    .col-md-12
+      - if @physical_tickets.present?
+        %table.table.table-bordered.table-striped.table-hover#roles
+          %thead
+            %th ID
+            %th Type
+            %th User
+            %th Actions
+          %tbody
+            - @physical_tickets.each do |physical_ticket|
+              %tr
+                %td= physical_ticket.id
+                %td= physical_ticket.ticket.title
+                %td= physical_ticket.user.name
+                %td
+                  .btn-group
+                    = link_to 'Show',
+                    conference_physical_ticket_path(@conference.short_title, physical_ticket.id),
+                    class: 'btn btn-primary'
+                    = link_to 'Generate PDF',
+                    conference_physical_ticket_path(@conference.short_title, physical_ticket.id,format: :pdf),
```

Comment:
you could write this as

```
conference_physical_ticket_path(@conference.short_title, 
                                physical_ticket.id,format: :pdf),
```

to avoid it too be that long :wink: